packages/shared_preferences/shared_preferences_platform_interface/README.md

```diff
@@ -1,17 +1,26 @@
 # shared_preferences_platform_interface
 
-A common platform interface for the [`shared_preferences`][1] plugin.
+A common platform interface for the [`shared_preferences`][1] plugins.
```

Comment:
It's still just one plugin; plugin is shorthand for plugin package, so refers to the whole package rather than the classes.